README.md

```diff
@@ -51,30 +51,27 @@ and processor architecture listed below.
 
 * CentOS 7
 * CentOS 8
-* Debian 9
 * Debian 10
 * FreeBSD 11.4
 * FreeBSD 12.2
-* Ubuntu 16.04
 * Ubuntu 18.04
 * Ubuntu 20.04
 
 ### Supported database engine versions
 
-Operating System | MariaDB         | PostgreSQL
----------------- | ----------------| -----------
-CentOS 7         | 5.5             | 9.3 
-CentOS 8         | 10.3            | 10.14 
-Debian 9         | 10.1            | 9.6
-Debian 10        | 10.3            | 11.7
-FreeBSD 11.4     | 5.7 (#)         | 12.4
-FreeBSD 12.2     | 5.7 (#)         | 12.4
-Ubuntu 16.04     | 10.0            | 9.5
-Ubuntu 18.04     | 10.1            | 10.14
-Ubuntu 20.04     | 10.3            | 12.4
+Operating System | MariaDB | PostgreSQL | SQLite
+---------------- | --------| -----------|------------------------
+CentOS 7         | 5.5     | 9.3        | 3.7.17 (?)
+CentOS 8         | 10.3    | 10.14      | 3.34 (?)
+Debian 10        | 10.3    | 11.7       | 3.27.2 (?)
+FreeBSD 11.4     | 5.7 (#) | 12.4       | 3.32.3 (##)
+FreeBSD 12.2     | 5.7 (#) | 12.4       | 3.32.3 (##)
+Ubuntu 18.04     | 10.1    | 10.14      | 3.22.0 (?)
+Ubuntu 20.04     | 10.3    | 12.4       | 3.33.0 (?)
 
-#) FreeBSD uses MySQL, not MariaDB.
+(#) FreeBSD uses MySQL, not MariaDB.
```

Comment:
Both MariaDB and MySQL are available on FreeBSD.

Reply:
Yes, but it does not work to use MariaDB with Zonemaster. After the release I can look at it to see how we could resolve it.

Reply:
Are MySQL and MariaDB incompatible on FreeBSD?

Reply:
I tested to use MariaDB, but DBI::mysql requires that MySQL is installed and that cannot be installed at the same time as MariaDB.

Reply:
You mean DBD::mysql? I checked /usr/ports/databases/p5-DBD-mysql/Makefile and /usr/ports/databases/p5-DBD-MariaDB/Makefile, and both have USES mysql. Here is a fair amount of guessing and speculation: Could it be that both MySQL and MariaDB implement "mysql" in FreeBSD, that FreeBSD locks in the meaning of "mysql" to the first implementation of it that you install, and that you had MySQL installed before you tried using DBD::mysql and MariaDB together?

Reply:
I can try again at next test section. I did not make work last time.

Reply:
@mattias-p, in FreeBSD:
* Backend requires DBD-mysql
* DBD-mysql requires MySQL-client
* MariaDB-server requires MariaDB-client
* MariaDB-client and MySQL-client cannot both be installed

Reply:
Too bad. Dependencies can be a real mess. It was worth a shot though.